spec/issues/webhooks/settings.yaml

```diff
@@ -0,0 +1,89 @@
+x-paths:
+
+  setup_webhook:
+    post:
+      summary: Setup URL for issues webhooks
+      description: >-
+        This is Smartling API endpoint
+        Setup URL for receiving issues webhooks
+      tags:
```

Comment:
"Setup URL for receiving issues webhooks" -> "Setup URL to receive issue webhooks"?